src/control/cmd/daos_server/start.go

```diff
@@ -91,9 +91,15 @@ func (cmd *startCmd) configureLogging() error {
 	// unless it was explicitly set to debug via CLI flag.
 	applyLogConfig := func() error {
 		switch logging.LogLevel(cmd.config.ControlLogMask) {
+		case logging.LogLevelTrace:
```

Comment:
The new log level TRACE and NOTICE should be added in the comment section of the control_log_mask key of the file utils/config/daos_server.yml

Reply:
Ah, yes. I'll add those, thanks.

Reply:
Done.